core/src/test/java/com/google/cloud/sql/core/DefaultConnectionInfoCacheTest.java

```diff
@@ -65,18 +65,18 @@ public void teardown() {
   @Test
   public void testCloudSqlInstanceDataRetrievedSuccessfully() {
     TestDataSupplier instanceDataSupplier = new TestDataSupplier(false);
-    // initialize instance after mocks are set up
-    CloudSqlInstance instance =
-        new CloudSqlInstance(
-            "project:region:instance",
+    // initialize connectionInfoCache after mocks are set up
+    DefaultConnectionInfoCache connectionInfoCache =
+        new DefaultConnectionInfoCache(
+            "project:region:connectionInfoCache",
```

Comment:
should we keep `project:region:instance`?

Reply:
Yes, and ditto below.

Reply:
Fixed.